src/Nest/Search/Search/Highlighting/Highlight.cs

```diff
@@ -115,5 +148,13 @@ public class HighlightDescriptor<T> : DescriptorBase<HighlightDescriptor<T> ,IHi
 		public HighlightDescriptor<T> BoundaryCharacters(string boundaryCharacters) => Assign(a => a.BoundaryChars = boundaryCharacters);
 
 		public HighlightDescriptor<T> BoundaryMaxSize(int boundaryMaxSize) => Assign(a => a.BoundaryMaxSize = boundaryMaxSize);
```

Comment:
I realise this is not part of the changes here, but it looks like `boundary_max_size` should be `boundary_max_scan`:

https://github.com/elastic/elasticsearch/blob/4863a9cfdf2ca5a2fdb503e5dab9372a0462cd20/core/src/main/java/org/elasticsearch/search/fetch/subphase/highlight/AbstractHighlighterBuilder.java#L64-L64

I think it makes sense to update as part of this PR